### ISSUE TITLE
Why is it important for POs and not scrum masters to create issues in backlog

### ISSUE THREAD

**Issue by iteles** [dwyl/process-handbook#54]
I spend a lot of time preaching that Product Owners should create issues rather than dwylers.

We need to make this clear in this repo and our processes. 

Why is this important?
+ Fosters a clearer sense of ownership of the backlog
+ Allows us to pick up on nuances and possible communication issues early (e.g. expectations that weren't clearly communicated verbally, incorrect understandings of conversations had over other channels)
+ Engages POs with Github early in the process and ensures they know to go there for updates rather than emailing members of the team (for example)
+ They learn from being asked the same questions a few times by developers and improve their creation of issues over time
+ We have clear criteria for what an issue is which surprisingly, we sometime forget to follow ourselves - POs are often better at following these if the correct expectation is set up front
+ Scrum masters' time is much _much_ better allocated to asking questions and unblocking the team than to writing up user stories which then still need clarification and validation from the PO

**Comment by Cleop** [comment]
Shall we move this one to the PO repo @iteles ?

**Comment by iteles** [comment]
@Cleop Yes, though I think it deserves a mention & link from this repo to the relevant section in the PO repo as it comes up a _lot_ for scrum masters as a question.
